pdns/pdns_recursor.cc

```diff
@@ -4228,6 +4228,7 @@ static void setupNODGlobal()
   g_nodLookupDomain = DNSName(::arg()["new-domain-lookup"]);
   g_nodLog = ::arg().mustDo("new-domain-log");
   parseNODWhitelist(::arg()["new-domain-whitelist"]);
+  parseNODWhitelist(::arg()["new-domain-ignore-list"]);
```

Comment:
We should like change the function names as well :+1:

Reply:
> We should like change the function names as well 👍

Agreed, I concentrated on the externally visible words first.